app/src/main/java/com/example/cs465/decisionhelper/dh_addfactors_textinput.java

```diff
@@ -48,12 +53,15 @@ private void addFactorsToView() {
             textView.setTextSize(26);
             textView.setText(f.name + ": ");
             textView.setHeight(height);
+            textView.setMaxWidth(width/2);
```

Comment:
You should set the maxwidth on the layout that contains all of the textviews instead of on each one.